programs/test/udp_proxy.c

```diff
@@ -523,11 +531,40 @@ void print_packet( const packet *p, const char *why )
     fflush( stdout );
 }
 
+/*
+ * In order to test the server's behaviour when receiving a ClientHello after
+ * the connection is established (this could be a hard reset from the client,
+ * but the server must not drop the existing connection before establishing
+ * client reachability, see RFC 6347 Section 4.2.8), we memorize the first
+ * ClientHello we see (which can't have a cookie), then replay it after the
+ * first ApplicationData record - then we're done.
+ *
+ * This is controlled by the inject_clihlo option.
+ *
+ * We want an explicit state and a place to store the packet.
+ */
+static enum {
```

Comment:
Please make this a typedef in case we want to declare a variable of this type

Reply:
I'm not sure I understand. We are declaring a variable of this type. Do you mean in case we want to declare another variable of this type? IMO that's rather unlikely to happen.

Reply:
Yes, for example if we want to write an auxiliary function. I think we shouldn't define a variable whose type cannot be named.